client/src/js/AppRouter.jsx

```diff
@@ -61,11 +62,30 @@ class AppRouter extends React.Component {
                 <AgreementContainer user={user} agreementId={props.match.params.id} />
               )}
             />
+            <Route
+              path='/admin/agreements' exact render={() => (
```

Comment:
What happens if we don't check `isAdmin`? will any user be able to see the list of agreements? Or the db permission rules would prevent that? I believe it's the latter, but I wonder if you actually tried it.

Reply:
the DB permission will actually prevent that, but the user shouldn't get to that page